### PR TITLE
feat: [FFM-4597]: wording, button, capitalisation improvements

### PR DESCRIPTION
**Pull request by crystal-fong**
### Summary

- Reworded the blurb under Flag Pipeline to enforce to users to meet the pre-reqs before attempting to add a Flag Pipeline
- Capitalised any Harness nouns that were missing that
- Reworded the `Add Flag Pipeline` to `+ Flag Pipeline` to meet UX standardisation efforts

#### Screenshots

Before | After 
--- | ---
image | image
image | image

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 #### 1 Issues 
* Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11422)

File: src/modules/75-cf/strings/strings.en.yaml [hunk 1]
```diff
@@ -244,28 +244,26 @@ featureFlags:
...
     noDataDescription: Flag Pipelines help to automate routine processes like approval requests, notifications, Jira ticket creation or update
-    noDataButtonText: Add Flag Pipeline
     drawerTitle: Flag Pipelines
-    drawerDescription:
-      Flag pipelines allow to automate routine actions and execute automatically after any change. Flag pipelines
-      have no CI or CD steps, instead they have only one flag change step. Flag, target and environment are all set
-      to runtime input
+    drawerDescription: 'You can only set Pipelines that:
+      <li> Have a Flag stage configured
+      <li> Don’t include a build or deploy stage'
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:259]: Looks like you've got list items, but no surrounding list
```suggestion
    drawerDescription: |
      You can only set Pipelines that:
      <ul>
        <li>Have a Flag stage configured</li>
        <li>Don’t include a build or deploy stage</li>
      </ul>
```

(note the pipe in the first line just tells YAML that multi-line text is coming and don't try to evaluate as anything else

Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:259]: You should also state that the _Flag_ must be set as a runtime input

File: src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/components/available-pipelines-drawer/AvailablePipelinesDrawer.tsx [hunk 2]
```diff
@@ -159,7 +159,7 @@ const AvailablePipelinesDrawer: React.FC<AvailablePipelinesDrawerProps> = ({
           />
         </Container>
         <Text font={{ variation: FontVariation.BODY2 }} color={Color.GREY_500}>
-          {getString('cf.featureFlags.flagPipeline.drawerDescription')}
+          <String stringID="cf.featureFlags.flagPipeline.drawerDescription" useRichText />
```
Comment (knagurski) [inline src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/components/available-pipelines-drawer/AvailablePipelinesDrawer.tsx:162]: Unfortunately a `<p>` tag doesn't allow things like lists inside it (it'll render, but it won't be happy about it). You should probably tell the `Text` component to be a `<div>` instead
```suggestion
        <Text font={{ variation: FontVariation.BODY2 }} color={Color.GREY_500} tag="div">
          <String stringID="cf.featureFlags.flagPipeline.drawerDescription" useRichText />
```

Comment (crystal-fong) [reply src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/components/available-pipelines-drawer/AvailablePipelinesDrawer.tsx:162]: huh okay didn't know that, updated

File: src/modules/75-cf/strings/strings.en.yaml (hunk 1 above)
Comment (crystal-fong) [reply src/modules/75-cf/strings/strings.en.yaml:259]: cool, looks neater and easier to read too

Comment (knagurski) [reply src/modules/75-cf/strings/strings.en.yaml]: Oh, remember to capitalise _Build_ and _Deploy_ as Harness nouns

Comment (crystal-fong) [reply src/modules/75-cf/strings/strings.en.yaml]: AH I considered it - feel like we need a dictionary of Harness official nouns

File: src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/components/available-pipelines-drawer/AvailablePipelinesDrawer.tsx [hunk 3]
```diff
@@ -158,8 +158,8 @@ const AvailablePipelinesDrawer: React.FC<AvailablePipelinesDrawerProps> = ({
             aria-label={getString('close')}
           />
         </Container>
-        <Text font={{ variation: FontVariation.BODY2 }} color={Color.GREY_500}>
-          {getString('cf.featureFlags.flagPipeline.drawerDescription')}
+        <Text font={{ variation: FontVariation.BODY2 }} color={Color.GREY_500} tag="div">
```
Comment (laura-mannus) [inline src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/components/available-pipelines-drawer/AvailablePipelinesDrawer.tsx:161]: isn't this `tag="div"` fab, can't believe I have only just started using it! nICE

File: src/modules/75-cf/strings/strings.en.yaml [hunk 4]
```diff
@@ -244,28 +244,30 @@ featureFlags:
...
+        <ul>
     drawerButtonText: Set as Flag Pipeline
     envText: 'Set all Flags in {{ env }}'
     saveSuccess: Flag Pipeline updated
     noAvailablePipelinesMessage: There are no Pipelines available
     noAvailablePipelinesDescription: You can create a Flag Pipeline in Pipeline Studio to automate your workflow
-    noExecutionMessage: There are no pipeline executions yet
-    noExecutionDescription: Once a pipeline has been triggered, the execution details will appear here
+    noExecutionMessage: There are no Pipeline executions yet
+    noExecutionDescription: Once a Pipeline has been triggered, the execution details will appear here
```
Comment (laura-mannus) [inline src/modules/75-cf/strings/strings.en.yaml:270]: You love yer capitalisations don't ye

**Review by laura-mannus — DISMISSED** [review]
Love it

**Comment by laura-mannus** [comment]
@crystal-fong there is a jest test failing may be because of a rebase?

**Comment by crystal-fong** [comment]
retrigger jest

File: src/modules/75-cf/strings/strings.en.yaml (hunk 4 above)
Comment (crystal-fong) [reply src/modules/75-cf/strings/strings.en.yaml:270]: caps aff - says me never

File: src/modules/75-cf/strings/strings.en.yaml [hunk 5]
```diff
@@ -244,28 +244,30 @@ featureFlags:
...
-    drawerDescription:
-      Flag pipelines allow to automate routine actions and execute automatically after any change. Flag pipelines
-      have no CI or CD steps, instead they have only one flag change step. Flag, target and environment are all set
-      to runtime input
+    drawerDescription: |
+      You can only set Pipelines that:
+        <ul>
+          <li> Have a Flag stage configured
+          <li> The Flag must be set as a runtime input
+          <li> Don’t include a Build or Deploy stage
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:262]: Oh come on. At least use valid HTML
```suggestion
          <li>Have a Flag stage configured</li>
          <li>The Flag must be set as a runtime input</li>
          <li>Don’t include a Build or Deploy stage</li>
```

Comment (crystal-fong) [reply src/modules/75-cf/strings/strings.en.yaml:262]: woops

**Review by knagurski — APPROVED** [review]
So. Dang. Good!

**Review by laura-mannus — APPROVED** [review]

**Pull request merged**